dataset.py

```diff
@@ -137,3 +142,91 @@ def yearly_255_trees_partitioned(reference_csv_filename: str) -> PartitionedData
   print_split(partition_data)
   return partition_data
 
+#Utility function for randomly sampling a point around a sample site
+def is_valid_point(lat: float, lon: float, reference_isocape: raster.AmazonGeoTiff):
+  return True if raster.get_data_at_coords(reference_isocape, lon, lat, 0) else False
```

Comment:
If the reason to change get_data_at_cords to not raise the exception, there are other ways to do this without changing it i.e.

try:
   _ = raster.get_data_at_coords(reference_isoscape, lon, lat, 0)
  return True
except ValueError:
   return False